src/main/resources/templates/ingredients/index.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html lang="en" xmlns:th="http//www.thymeleaf.org">
-<head th:replace="fragments :: head>
+<head th:replace="fragments :: head">
```

Comment:
Your whole template for this page wasn't loading 'cuz you forgot to close this string.